packages/react-components/cypress/support/commands.ts

```diff
@@ -0,0 +1,47 @@
+import { mount } from '@cypress/react'
+
+import { resetKnobs } from '../utils/storybook'
+
+function stopVideo(elements: JQuery<HTMLVideoElement> | void) {
```

Comment:
@pshoniuk is this for Percy? so we can stop the video and then take a screenshot?